docs/tutorials/api/datasets.ipynb

```diff
@@ -0,0 +1,1163 @@
+{
```

Comment:
<p>Similar here, I think this documentation rather belongs to the <code>EDispKernelMap</code>  and <code>gammapy.irf</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>